components/script/dom/element.rs

```diff
@@ -2575,6 +2616,28 @@ impl Element {
         self.set_click_in_progress(false);
     }
 
+    pub fn get_lang(&self) -> Atom {
+        if let Some(ref lang) = *self.cached_lang.read() {
+            lang.clone()
```

Comment:
The WPT tests are still giving me TIMEOUT errors, and I suspect it's something to do with being deadlocked. Can somebody confirm that the temporary `RwReadGuard` is dropped immediately after this conditional?